lib/std/zig/parser_test.zig

```diff
@@ -5439,9 +5483,9 @@ test "ampersand" {
     });
 
     try testError(
-        \\ _ = bar & &foo, 
-        \\ _ = bar & &&foo, 
-        \\ _ = &&foo, 
```

Comment:
As for these trailing whitespace removals, my editor did that and these changes are unrelated. But I thought, while I was at it, why not remove some useless trailing whitespace? I hope they are useless, at least.
If this should be done in another PR, please tell me.